sdk/webpubsub/azure-messaging-webpubsub/src/main/java/com/azure/messaging/webpubsub/WebPubSubServiceClient.java

```diff
@@ -39,7 +42,7 @@ public final class WebPubSubServiceClient {
      * @param serviceClient the service client implementation.
      */
     WebPubSubServiceClient(WebPubSubsImpl serviceClient, String hub, String endpoint,
-        AzureKeyCredential keyCredential) {
+                           AzureKeyCredential keyCredential) {
```

Comment:
Better to avoid these unnecessary formatting.

Reply:
These are actually automatically generated when I run Autorest. Should I undo it instead?